openedx/stanford/djangoapps/auth_lagunita/models.py

```diff
@@ -36,6 +36,10 @@ class Info(models.Model):
         auto_now=True,
     )
 
+    class Meta(object):
+        app_label = 'openedx.stanford.djangoapps.auth_lagunita'
```

Comment:
is there no `auth_lagunita` tests file?

Reply:
This isn't strictly for tests; it's for the djangoapp.